src/components/Uploads/UploadButton.js

```diff
@@ -35,6 +36,12 @@ const StyledUploadButton = styled.button`
   }
 `;
 
+const StyledIcon = styled.img`
+  position: relative;
+  width: ${props => props.width || '24px'};
+  height: ${props => props.height || '24px'};
```

Comment:
Looks like `props.height` and `props.width` are never used?
Also I don't think `position: relative` does something here? Since it has no children with absolute position.

If it's not used, let's remove it.

Reply:
Ok, I'll  fix it)